src/modules/commander/commander_params.c

```diff
@@ -1059,3 +1059,95 @@ PARAM_DEFINE_FLOAT(COM_THROW_SPEED, 5);
  * @increment 1
  */
 PARAM_DEFINE_INT32(COM_FLTT_LOW_ACT, 3);
+
+/**
+ * It is the color that indicates that the mission is suitable.
```

Comment:
I would recommend taking a look at this in QGC, the first line is a short description so things like "Mission suitable LED color" work well. Then if needed you can add a long description after.

Reply:
I gonna change the description of the parameters. Description has to be more understandable.